### PR TITLE
Adding from_index_startswith builds search filter

### PR DESCRIPTION
**Pull request by lipoja**
This is required for monitoring script to easily search requests for index image without specifying tag

### CODE REVIEW

**Comment by zanssa** [comment]
@lipoja Shouldn't we add test functions for testing the filter?

File: iib/web/api_v1.py [hunk 1]
```diff
@@ -432,6 +433,30 @@ def get_builds() -> flask.Response:
                 )
             )
 
+        if from_index_startswith:
+            query_params['from_index_startswith'] = from_index_startswith
+            from_index_results = Image.query.filter(
```
Comment (zanssa) [inline iib/web/api_v1.py:438]: should this be `from_index_results` or `from_index_startswith_results `?

Comment (lipoja) [reply iib/web/api_v1.py:438]: yeah it would be better :)

Comment (lipoja) [reply iib/web/api_v1.py]: please review changes

**Review by xDaile — APPROVED** [review]

**Review by chandwanitulsi — APPROVED** [review]

**Review by zanssa — APPROVED** [review]

File: iib/web/api_v1.py (hunk 1 above)
Comment (zanssa) [reply iib/web/api_v1.py]: Thank you!

**Pull request merged**